prometheus_client/twisted.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
```

Comment:
This would probably be best to be in a twisted package, in a file called exposition.py. There may be other twisted related functionality in the future.